### ISSUE TITLE
Propose a new diffChecker feature for the newly created Visual Git Editor

### ISSUE THREAD

**Issue by karthiksriramuoa** [kblincoe/VisualGit_SE701_3#133]
**_Feature Request_**

A diff checker is a very useful feature when comparing the same file to resolve conflicts and manage the changes made by different team members. This is particularly useful when the file is significantly large and there are multiple modules that the team members are working on. 

Scenario -

Suppose a complex security module is assigned to a team of 5 security engineers and each of them are working on different aspects of the same feature. When 2 or more engineers are trying to commit and want to check if there are any dependencies on other's code that will impact on the current commit, it will be great if one can do that without closing the current file editor.

It will be a good addition to the existing file editor.

@softeng-701 requesting for approval

**Comment by softeng-701** [comment]
@karthiksriramuoa Do you intend to work on this? I'm not sure there is time to complete this...

**Comment by karthiksriramuoa** [comment]
No, I don't have time to work on this. But just proposing a feature request which could be done.

**Comment by softeng-701** [comment]
@karthiksriramuoa  Approved
